app/assets/javascripts/shared/behaviors.js

```diff
@@ -81,11 +81,21 @@
     }
 
     // Update address bar with current tab param
-    $('[data-bs-toggle~=tab]').on('shown.bs.tab', function (e) {
-      let currentTab = $(e.target).attr('href').substring(1);
-      searchParams.set('tab', currentTab);
-      history.pushState(null, null, `?${searchParams.toString()}`);
-    });
+    $('[data-bs-toggle~=tab]')
+      .unbind()
```

Comment:
It's mentioned here: https://api.jquery.com/unbind/ that the method is now deprecated in favor of `off`